src/components/radio/radio.ts

```diff
@@ -71,8 +71,8 @@ export class MdRadioGroup implements AfterContentInit, ControlValueAccessor {
   /** The currently selected radio button. Should match value. */
   private _selected: MdRadioButton = null;
 
-  /** Change event subscription set up by registerOnChange (ControlValueAccessor). */
-  private _changeSubscription: {unsubscribe: () => any} = null;
+  /** The method to be called in order to update ngModel */
+  private _onChangeHandler: (value: number) => void = (value) => {};
```

Comment:
What do you think of renaming this to `_controlValueAccessorChangeFn`? (similar for the touch one as well)